airmozilla/popcorn/templates/popcorn/editor.html

```diff
@@ -23,6 +23,7 @@
 {% endblock %}
 
 {% block full_width_content %}
+<slug data-url="{{ slug }}"></slug>
 <div id="editor"></div>
```

Comment:
You could put that `data-url={}` thing on the `<div id="editor">` element which you already have.